docs/installation.md

```diff
@@ -0,0 +1,69 @@
+# Installing CodeSee in your project
+
+Following the instructions below, you will:
+
+1. Install two npm packages.
+2. Configure Babel so that CodeSee instruments your code when running in development.
+3. Rebuild and run your app locally.
+
+
+## Step 1: Install the CodeSee package in your JavaScript or TypeScript app
+In the terminal, run the install command for your package manager:
+
+<details><summary>npm</summary>
+
+```
+npm install --save-dev @codesee/tracker@latest @codesee/babel-plugin-instrument@latest
+```
+
+</details>
+<details><summary>yarn</summary>
+
+```
+yarn add --dev @codesee/tracker@latest @codesee/babel-plugin-instrument@latest
+```
+</details>
+
+## Step 2: Configure your project for CodeSee
+Follow the links below to set up CodeSee for your specific project environment:
+
+- <a href="/setup-cra" target="_blank">**Create React App**</a>
+- <a href="/setup-typescript-without-babel" target="_blank">**Typescript, but without Babel**</a>
+- <a href="/setup-nuxt" target="_blank">**Nuxt**</a>
+- <a href="/setup-storybook" target="_blank">**Storybook**</a>
+- <a href="/setup-gatsby" target="_blank">**Gatsby**</a>
+- <a href="/setup-ember" target="_blank">**Ember**</a>
+
+Or, select generic setup if you use Next.js, Typescript with Babel, or another framework not listed above:
+
+- <a href="/setup-generic" target="_blank">**Generic setup**</a>
```

Comment:
Do we know these links will work, or will we need to try that out?

Reply:
Adding the step back to this page at the end of each of these sub pages, so will change these to markdown links that open in the same window.